resume-buddy/src/main/java/com/google/sps/servlets/DeleteBlobServlet.java

```diff
@@ -1,32 +0,0 @@
-import com.google.appengine.api.blobstore.BlobInfoFactory;
```

Comment:
Looks like a bad sync deleted this file.

Reply:
I wanted to call this servlet from deleteData servlet, but since it separately does not need to be a servlet, I moved the code into deleteBlob function